src/main/java/de/tum/in/www1/artemis/service/plagiarism/ContinuousPlagiarismControlService.java

```diff
@@ -49,7 +57,10 @@ public void executeChecks() {
             PlagiarismChecksConfigHelper.createAndSaveDefaultIfNull(exercise, exerciseRepository);
 
             try {
-                executeChecksForExercise(exercise);
+                var result = executeChecksForExercise(exercise);
+                if (exercise.getExerciseType() != ExerciseType.PROGRAMMING) {
```

Comment:
Remove this part as soon as the problem with the programming exercises is fixed.

Reply:
That will be my next pr